src/osp/Active/SysVehicle.cpp

```diff
@@ -107,24 +103,25 @@ ActiveEnt SysVehicle::activate(ActiveScene &rScene, universe::Universe &rUni,
     rScene.get_registry().reserve(rScene.get_registry().capacity()
                                    + vehicleData.m_machines.size());
 
-    // Keep track of parts
+    // Keep track of parts                rScene.reg_emplace<ACompFloatingOrigin>(islandEnt);
```

Comment:
rScene.reg_emplace<ACompFloatingOrigin>(isLandEnt);
is commented out.

If that was intentional, please change the comment to make it more obvious that it was intentional.